core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -278,6 +278,17 @@
                     }
                 }
             }
+
+            .ai-loading {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                min-width: 100%;
+                padding: 0.5rem;
```

Comment:
what about using the variables for spacing?

Reply:
done